### PR TITLE
adding changes to GA4 topics

### PR DESCRIPTION
**Pull request by cmastr**
https://segment.atlassian.net/browse/DOC-840

### CODE REVIEW

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 1]
```diff
@@ -8,65 +8,67 @@ versions:
...
+Google Analytics 4 advises for the use of [Recommended events](https://support.google.com/analytics/answer/9267735){:target="_blank"} and properties to power certain built-in reports. Segment’s Google Analytics 4 Web destination provides prebuilt mappings to automatically map your Segment spec events to the corresponding Google Analytics 4 events and properties. If your Segment events don’t follow the Segment spec exactly, you can modify the mappings. For example, Segment maps `Order Completed` events to the Google Analytics 4 `Purchase` event by default. If your company uses `Products Purchase` to indicate a purchase, you can map it in the Purchase action’s Event Trigger instead.
```
Comment (cmastr) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:41]:
```suggestion
Google Analytics 4 advises for the use of [Recommended events](https://support.google.com/analytics/answer/9267735){:target="_blank"} and properties to power certain built-in reports. Segment’s Google Analytics 4 Web destination provides prebuilt mappings to automatically map your Segment spec events to the corresponding Google Analytics 4 events and properties. If your Segment events don’t follow the Segment spec exactly, you can modify the mappings. or example, Segment maps `Order Completed` events to the Google Analytics 4 `Purchase` event when you add the `Purchase` action to your mappings. If your company uses `Products Purchase` to indicate a purchase, you can map it in the Purchase action’s Event Trigger instead.
```

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---
+## Google Analytics 4 FAQ and troubleshooting
+
+### Debug mode
+
+The Google Analytics 4 [debug mode](https://support.google.com/analytics/answer/7201382?hl=en){:target="_blank"} is supported with the Google Analytics 4 Web destination. DebugView displays the events and user properties that Analytics collects from a user in real-time. This can be helpful in troubleshooting your implementation.
+
+### Send events from both the browser and the server 
+
+With Google Analytics 4 Web, events are sent from the browser (client-side) to GA4. If you use Segment’s [Google Analytics 4 Cloud destination](/docs/connections/destinations/catalog/actions-google-analytics-4/#benefits-of-google-analytics-4-cloud) to send events through the Measurement Protocol API and want to tie data between client-side and server-side, you need to pass the same Client ID from the browser and the server for GA4 to merge events around a distinct user, and if you want the server-side event to be tied to the client-side session, you must pass the session_id in the server side event.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:14]: Edit:
```With Google Analytics 4 Web, events are sent from the browser (client-side) to GA4. If you use Segment’s [Google Analytics 4 Cloud destination](/docs/connections/destinations/catalog/actions-google-analytics-4/#benefits-of-google-analytics-4-cloud) to send events through the Measurement Protocol API and want to tie data between client-side and server-side, you need to pass the same Client ID from the browser and the server for GA4 to merge events around a distinct user.  Furthermore, if you want the server-side event to be tied to the client-side session, you must pass the session_id in the server side event.```

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+The client_id and the session_id are both cookies stored in the user browser. You can use [Google gtag get commands](https://developers.google.com/tag-platform/gtagjs/reference#get){:target=”_blank”} or other cookie methods to parse these values from the _ga cookie and _ga_measurementId cookie:
+If your _ga cookie is GA1.1.1783165678.1701112990 then 1783165678.1701112990 is your client_id
+If your _ga_M12454XDR cookie is GS1.1.1710342977.347.1.1710343074.0.0.0 then 1710342977 is you session_id
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:18]: Edit typo:
`If your _ga_M12454XDR cookie is GS1.1.1710342977.347.1.1710343074.0.0.0 then 1710342977 is your session_id`

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Data takes a long time to appear in Google's reports
+
+Google may take [24-48 hours](https://support.google.com/analytics/answer/9333790){:target="_blank"} to process data sent to Google Analytics. As a result, the Google Analytics user interface may not reflect the most current data. The Google Analytics [Realtime report](https://support.google.com/analytics/answer/9271392){:target="_blank"} displays activity on your site as it happens; however, events seen in Realtime Reports do not always equate to how the data is processed in the standard reports. Typically we see this when a Custom Definition is not defined in the GA4 Admin panel.
+
+### Data is not sent to Google
+
+For event data to be sent downstream to Google Analytics:
+
+1. The **Set Configuration Fields** mapping must be configured and enabled in your mappings. This mapping is required for data to be sent downstream because it sets configuration to the GA4 Measurement ID indicated in the Settings and establishes data flow using the `config` command.
+
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:37]: Remove space

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+2. Ensure that you're calling `analytics.page()` on page load. Analytics.js requires an initial Page call to send data to Google Analytics 4 Web. The Segment snippet includes this initial call by default.
+3. Send data with an event, typically this is a page_view as your first event. Note: If you toggled “On” Page Views in your Settings, then the page_view event will automatically send when Set Configuration Mapping is triggered. If you need to override this setting for your particular use case, read {{link to this section: ### Can I override my send_page_view selection that I declared in the Settings?}}
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:39]: Add link... my notes were a placeholder

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+All in all, the Set Configuration Mapping is a crucial part of your GA4 Web setup. It provides the “handshake” between your site and the gtag library. If no events are not sending to your GA4 instance, then use one of the Debugging Tools to check the sequence of GA4 events
+
+### Duplicate `page_view` events in GA4
+
+If you are sending multiple `gtag(‘config’)` commands called from Set Configuration mapping on one page before a new DOM has loaded, and you have defined `send_page_view: true` with each ‘Config’ event, you may see duplicate `page_view` events sent to your GA4 measurement id. . If this is the case, please refer to Google's documentation on [Ignoring duplicate instances of on-page configuration](https://support.google.com/analytics/answer/9973999?hl=en#:~:text=as%20described%20below.-,Ignore%20duplicate%20instances%20of%20on%2Dpage%20configuration,Click%20Save.,-Give%20feedback%20about){:target="_blank"}.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:45]: Typo after "events sent to your GA4 measurement id."  There are two periods.

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+### Tracking UTM Parameters
+
+UTM Parameters are automatically tracked and sent to Google when they are present in the URL. For example, with the following URL:
+`https://www.example.com/?utm_content=email_variation1&utm_medium=email&utm_source=email_promo&utm_campaign=summer_sale&utm_id=abcd`
+
+By default, if the UTM values are found in the `page_location` parameter, GA4 will automatically map the campaign parameters to their appropriate value. There is no additional configuration required, unless you want to override what GA4 defines. This is true for both Event mappings and the Set Configuration mapping. If you modify the output of the `page_location` and the UTM parameters are not included, this could result in erroneous acquisition and attribution mapping. Furthermore, you can observe by triggering a `Page` call with UTM parameters present in the URL and navigating to the **Realtime overview** report in GA4 to see the resulting `page_view` event under the _Event count by Event name_ card - the Acquisition card in Realtime reflects First User acquisition and may not reflect the parameters of the event if this was a second session event
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:63]: Typo, add period end of paragraph.

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 8]
```diff
@@ -8,65 +8,67 @@ versions:
...
 
-To connect the Google Analytics 4 Web destination: 
+To connect the Google Analytics 4 Web destination:
 
 1. From the Segment web app, click **Catalog**, then click **Destinations**.
 2. Search for “Google Analytics 4 Web” in the Destinations Catalog, and select the destination.
 3. Click **Configure Google Analytics 4 Web**.
-4. Select the web source that will send data to Google Analytics 4 and follow the steps to name your destination. The web source chosen must use [Analytics.js 2.0](/docs/connections/sources/catalog/libraries/website/javascript/).
+4. Select the web source that will send data to Google Analytics 4 and follow the steps to name your destination. The web source chosen must use [Analytics.js 2.0](/docs/connections/sources/catalog/libraries/website/javascript/). For mobile source tracking, view Firebase Destination configuration {{insert link}}.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:29]: insert link to Segment Firebase Destination

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 9]
```diff
@@ -8,65 +8,67 @@ versions:
...
 
-Some parameters automatically populate prebuilt [dimensions and metrics](https://support.google.com/analytics/answer/9143382){:target="_blank"} in Google Analytics. For example, the parameters on the automatically collected and enhanced measurement events and the required and optional parameters you send with the recommended events both populate pre-built dimensions and metrics.
+Some events automatically populate in Google Analytics. For example, there are some [Automatically Collected] (https://support.google.com/analytics/answer/9234069){:target=”_blank”} events collected by triggering the Set Configuration Mapping. For example, calling gtag(‘config’), and [Enhanced Measurement events](https://support.google.com/analytics/answer/9216061){:target=”_blank”} controlled by toggling “on” in your GA4 Admin panel use event listeners to send events; in addition, all events tracked via GA4 Web, populate some commonly used parameters like page_location without additional configuration. Review Google’s documents for more information found [here](https://support.google.com/analytics/table/13594742){:target=”_blank”}
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:37]: Edit using markdown to highlight, do not add ticks to final:
`Some events automatically populate in Google Analytics. For example, there are some [Automatically Collected] (https://support.google.com/analytics/answer/9234069){:target=”_blank”} events collected by triggering the Set Configuration Mapping. Additionally, calling gtag(‘config’), and enabling [Enhanced Measurement events](https://support.google.com/analytics/answer/9216061){:target=”_blank”}, which are controlled by toggling “on” in your GA4 Admin panel, will use event listeners to send events; moreover, all events tracked via GA4 Web, populate some commonly used parameters like page_location without additional configuration. Review Google’s documents for more information found [here](https://support.google.com/analytics/table/13594742){:target=”_blank”}`

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 10]
```diff
@@ -82,15 +84,15 @@ See [set up consent mode on websites](https://developers.google.com/tag-platform
 To enable consent mode for your Google Analytics 4 Web destination: 
 1. Navigate to **Connections > Destinations** and select your Google Analytics 4 Web** destination. 
 2. Go to the **Settings** tab of the destination.
-3. Click the toggle on for **Enable Consent Mode**. 
-4. Set the following fields with these values:
+3. Click the toggle on for **Enable Consent Mode**, this calls gtag(‘consent’,’default’) with the defined parameters when the gtag library loads.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:87]: I added a note not to modify this section.  I'm not confident in my knowledge of what changes were made; therefore, remove any edits from Consent Mode section.

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 11]
```diff
@@ -82,15 +84,15 @@ See [set up consent mode on websites](https://developers.google.com/tag-platform
...
+4. Set the following fields with your organization’s determination of granted or denied:
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:88]: Remove change

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 12]
```diff
@@ -82,15 +84,15 @@ See [set up consent mode on websites](https://developers.google.com/tag-platform
...
     Field | Value
     ----- | ------
-    Default Ad Storage Consent State | Granted
-    Default Analytics Storage Consent State | Granted
-    Ad User Data Consent State | Granted
-    Ad Personalization Consent State | Granted
+    Default Ad Storage Consent State | Granted or Denied
+    Default Analytics Storage Consent State | Granted or Denied
+    Ad User Data Consent State | Granted or Denied
+    Ad Personalization Consent State | Granted or Denied
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:95]: Remove changes on 92-95

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 13]
```diff
@@ -101,7 +103,7 @@ To enable consent mode for your Google Analytics 4 Web destination:
         'Analytics Storage Consent State': 'false'
     });
     ```
-7. As soon as the page loads and the set configuration fires to the Google Analytics SDK, Segment issues a consent mode update command. Map the properties you defined to collect consent state changes to the Set Configurations Fields mapping. You can choose to do this from 1 of 2 options in steps 4 and 5. 
+7. As soon as the page loads and the set configuration fires to the Google Analytics SDK, Segment issues a consent mode update command. Map the properties you defined to collect consent state changes to the Set Configurations Fields mapping. You can choose to do this from 1 of 2 options in steps 4 and 5.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:106]: Remove

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 14]
```diff
@@ -111,68 +113,14 @@ To enable consent mode for your Google Analytics 4 Web destination:
             * Ad User Data Consent State
             * Ad Personalization Consent State
             You can manually select `Granted` or `Denied` from the dropdown menu for Advanced consent mode settings, and type in `granted` or `denied` for basic consent mode settings.
-        5. *(Option 2)* Under the **Select mappings** section, create an event variable to directly grab the value from the payload (for example, `properties.adStorageConsentState`). Ensure it translates to `granted` or `denied`. You can use an insert function to translate to `granted` or `denied`. Use the replace function if it's a string. Do this for these fields:
+        5. *(Option 2)* Under the **Select mappings** section, create an event variable to directly grab the value from the payload. For example, `properties.adStorageConsentState`. Ensure it translates to `granted` or `denied`. You can use an insert function to translate to `granted` or `denied`. Use the replace function if it's a string. Do this for these fields:
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:116]: remove

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 15]
```diff
@@ -111,68 +113,14 @@ To enable consent mode for your Google Analytics 4 Web destination:
...
             * Ads Storage Consent State
             * Analytics Storage Consent State
             * Ad User Data Consent State
             * Ad Personalization Consent State
 
-When these properties are available, they send to the `update` command. 
+When these properties are available, they send to the `update` command.
```
Comment (jsabor) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:122]: remove

**Review by jsabor — CHANGES_REQUESTED** [review]
I added several comment for changes and there are two missing links.  Any changes I suggested for consent mode do not need to be included in this update.

**Review by forstisabella — CHANGES_REQUESTED** [review]
A couple of changes and I think we'll be good to go. If you have any questions about my suggestions, let me know! @rvadera12

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+
+1. The **Set Configuration Fields** mapping must be configured and enabled in your mappings. This mapping is required for data to be sent downstream because it sets configuration to the GA4 Measurement ID indicated in the Settings and establishes data flow using the `config` command.
+
+2. Ensure that you're calling `analytics.page()` on page load. Analytics.js requires an initial Page call to send data to Google Analytics 4 Web. The Segment snippet includes this initial call by default.
+3. Send data with an event, typically this is a page_view as your first event. Note: If you toggled “On” Page Views in your Settings, then the page_view event will automatically send when Set Configuration Mapping is triggered. If you need to override this setting for your particular use case, read {{link to this section: ### Can I override my send_page_view selection that I declared in the Settings?}}
+
+All in all, the Set Configuration Mapping is a crucial part of your GA4 Web setup. It provides the “handshake” between your site and the gtag library. If no events are not sending to your GA4 instance, then use one of the Debugging Tools to check the sequence of GA4 events
```
[the remark this answers is not in the document]
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:41]: Both - you can use the google debugging tools or our event delivery

**Review by forstisabella — APPROVED** [review]

File: src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+
+ > note "If you toggled Page Views in your Settings to “On”, the page_view event automatically sends when the Set Configuration Mapping is triggered"
+ > If you need to override this setting for your particular use case, see [Can I override my send_page_view selection that I declared in Settings?](#can-i-override-my-send_page_view-selection-that-i-declared-in-settings)
+
+If no events are flowing to your GA4 instance, use one of the Debugging Tools to check the sequence of GA4 events.
+
+## Duplicate `page_view` events in GA4
+
+If you are sending multiple `gtag(‘config’)` commands called from Set Configuration mapping on one page before a new DOM has loaded, and you have defined `send_page_view: true` with each ‘Config’ event, you may see duplicate `page_view` events sent to your GA4 measurement id. . If this is the case, please refer to Google's documentation on [Ignoring duplicate instances of on-page configuration](https://support.google.com/analytics/answer/9973999?hl=en#:~:text=as%20described%20below.-,Ignore%20duplicate%20instances%20of%20on%2Dpage%20configuration,Click%20Save.,-Give%20feedback%20about){:target="_blank"}.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/ga4-faq-and-troubleshooting:46]:
```suggestion
If you are sending multiple `gtag(‘config’)` commands called from Set Configuration mapping on one page before a new DOM has loaded, and you have defined `send_page_view: true` with each ‘Config’ event, you may see duplicate `page_view` events sent to your GA4 measurement id. If this is the case, please refer to Google's documentation on [Ignoring duplicate instances of on-page configuration](https://support.google.com/analytics/answer/9973999?hl=en#:~:text=as%20described%20below.-,Ignore%20duplicate%20instances%20of%20on%2Dpage%20configuration,Click%20Save.,-Give%20feedback%20about){:target="_blank"}. 
```

**Pull request merged**